turbinia/turbiniactl.py

```diff
@@ -861,6 +982,17 @@ def main():
               region=region, request_id=request.request_id,
               all_fields=args.all_fields))
 
+
+def main():
+  """Main function for turbiniactl"""
+  # TODO(aarontp): Allow for single run mode when
```

Comment:
I think this comment can be removed now.

Reply:
Done